### PR TITLE
Fingerprint(STR-480)

### PR DESCRIPTION
**Pull request by saito-sv**
## The event is being handled and data is being persisted on eventManager props.

unity SDK

### CODE REVIEW

File: Runtime/StringEvents.cs [hunk 1]
```diff
@@ -26,6 +26,13 @@ public class VendorChangedPayload
     public bool accepted;
 }
 
+[Serializable]
```
Comment (auroter) [inline Runtime/StringEvents.cs:29]: This is already defined in Types.cs

Comment (saito-sv) [reply Runtime/StringEvents.cs:29]: ill leave that up to you.

**Review by auroter — APPROVED** [review]
LGTM

File: Runtime/WebEventManager.cs [hunk 2]
```diff
@@ -98,6 +100,15 @@ private static void parseEventPayload(string strPayload)
                 CardValid = validation.data.data.valid;
                 CardValidationChanged?.Invoke(validation);
                 break;
+            case "fingerprint":
+                var fingerprint = StringEvent<FingerprintPayload>.FromJSON(strPayload);
+                 if (fingerprint == null) 
+                 { 
+                    break;
```
Comment (alcalawil) [inline Runtime/WebEventManager.cs:107]: our API accepts fingerprint empty values, so we allow the user to transact with an unknown device
```suggestion
                FingerprintVisitorId = ""
                FingerprintRequestId = ""
```

Comment (saito-sv) [reply Runtime/WebEventManager.cs:107]: These values are already empty strings if we don’t set them here

**Review by alcalawil — APPROVED** [review]

**Pull request merged**